modules/combineReducers.js

```diff
@@ -12,32 +12,36 @@ function optimizeBatch(effects) {
   }
 }
 
-export function combineReducers(reducerMap) {
-  return function finalReducer(state = {}, action) {
-    let hasChanged = false;
-    let effects = [];
-    let model = {};
+export function combineReducers(reducerMap, rootState = {}, accessor, mutator) {
+    if (typeof accessor !== 'function') {
```

Comment:
can you lift the default `accessor` and `mutator` functions out of the `combineReducers` body and use default args to assign them? could be a little bit of a perf boost to not create new default functions on every call